kfdef/kfdef.yaml

```diff
@@ -15,15 +15,15 @@ spec:
           - name: IMAGES_SCHEDULEDWORKFLOW
             value: quay.io/opendatahub/ds-pipelines-scheduledworkflow:main
           - name: IMAGES_CACHE
-            value: registry.access.redhat.com/ubi8/ubi-minimal:8.7
+            value: registry.access.redhat.com/ubi8/ubi-minimal:8.8
           - name: IMAGES_DSPO
             value: quay.io/opendatahub/data-science-pipelines-operator:main
           - name: IMAGES_MOVERESULTSIMAGE
-            value: registry.access.redhat.com/ubi8/ubi-micro:8.7
+            value: registry.access.redhat.com/ubi8/ubi-micro:8.8
           - name: IMAGES_MARIADB
             value: registry.redhat.io/rhel8/mariadb-103:1-188
           - name: IMAGES_OAUTHPROXY
-            value: registry.redhat.io/openshift4/ose-oauth-proxy:v4.12.0
+            value: registry.redhat.io/openshift4/ose-oauth-proxy:v4.13.0
```

Comment:
is oauth-proxy also impacted by this?

Reply:
Not really. I saw ose-cli and imagined that ose-auth-proxy should be updated as well.

Reply:
Reverted. Thank you.